src/container/srv_container.c

```diff
@@ -854,7 +854,43 @@ cont_open(struct rdb_tx *tx, struct ds_pool_hdl *pool_hdl, struct cont *cont,
 	D_ASSERT(prop != NULL);
 	D_ASSERT(prop->dpp_nr == CONT_PROP_NUM);
 
-	if (!(in->coi_capas & DAOS_COO_FORCE)) {
```

Comment:
there is a comment above "Determine pool meets container redundancy factor requirements". I think that should move down to the code following the insertion of this security enforcement-related code.

Reply:
Fixed